sections/staking/components/StakingInput/StakingInput.tsx

```diff
@@ -59,7 +66,9 @@ type StakingInputProps = {
 	setTransactionState: (tx: Transaction) => void;
 	maxBurnAmount?: BigNumber;
 	burnAmountToFixCRatio?: BigNumber;
-	canClearDebt?: boolean;
+	etherNeededToBuy?: string;
```

Comment:
i believe these are the wrong types, since you are still returning ```null``` in ```useClearDebtCalculations``` can either can it to return undefined or include the null type and remove the ? 

i.e 

```sUSDNeededToBuy: string | null```

Reply:
maybe I'm wrong but I believe the only potential null return from `useClearDebtCalculations` is swapData

etherNeededToBuy, sUSDNeededToBuy, etc.. should all be a value or undefined
https://github.com/Synthetixio/staking/pull/366/files#diff-90794eba7d469521083225febf4e4e61c50f0cf62ce07af55793f70670110d27R296